From 3f3dbb5e4a174ac2ae5592bdb84cba19e3d4506b Mon Sep 17 00:00:00 2001
From: Isabel Paredes <isabel.paredes@quantstack.net>
Date: Mon, 17 Nov 2025 17:33:41 +0100
Subject: [PATCH 2/2] Fix duplicate symbols

Library pil_imaging_mode is added twice causing duplicate symbol errors.
---
 setup.py | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/setup.py b/setup.py
index 178dad9..153172b 100644
--- a/setup.py
+++ b/setup.py
@@ -1087,8 +1087,8 @@ for src_file in _IMAGING:
 for src_file in _LIB_IMAGING:
     files.append(os.path.join("src/libImaging", src_file + ".c"))
 ext_modules = [
-    Extension("PIL._imaging", files, libraries=["pil_imaging_mode"]),
-    Extension("PIL._imagingft", ["src/_imagingft.c"], libraries=["pil_imaging_mode"]),
+    Extension("PIL._imaging", files),
+    Extension("PIL._imagingft", ["src/_imagingft.c"]),
     Extension("PIL._imagingcms", ["src/_imagingcms.c"], libraries=["pil_imaging_mode"]),
     Extension("PIL._webp", ["src/_webp.c"], libraries=["pil_imaging_mode"]),
     Extension("PIL._avif", ["src/_avif.c"]),
